FrendsTaskTemplate/README.md

```
@@ -0,0 +1,94 @@
+
+You can create a new Task by running the following command in the Task's repository folder.
+
+`dotnet new frends-task -F Frends.ClassName.MethodName -D "Description of the Task"`
+
+## Get help using the template
+
+`dotnet new frends-task -h`
+
+### Example output:
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)